tiers/checks.go

```diff
@@ -122,22 +121,22 @@ func userEditRoleBinding() innerObjectCheck {
 	}
 }
 
-func toolchainUserEditRole(roleName string) innerObjectCheck {
+func rbacEditRoleBinding() innerObjectCheck {
```

Comment:
wouldn't it be more flexible if we keep the resource name as a parameter of these funcs?

Reply:
Since we verify also the content of the resources, then I don't think that we could reuse it - I don't expect that we would have multiple resources with the same content but with different names.

Reply:
And that is exactly what I'm trying to fix in these two PRs - have the resource names consistent

Reply:
ok